KsApi/models/graphql/adapters/User+UserFragment.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+extension User {
```

Comment:
Maybe add a `TODO:// Remove file once all instances of this function are replaced by User+UserFragment` in `User+GraphUser`

Reply:
There will be a clean-up task to remove all old Graph* files once the Apollo integration is done.